Source/LuaBridge/RefCountedObject.h

```diff
@@ -154,7 +151,7 @@ class RefCountedObjectPtr
 
     //==============================================================================
     /** Creates a pointer to a null object. */
-    inline RefCountedObjectPtr() : referencedObject(0) {}
+    inline RefCountedObjectPtr() : referencedObject(nullptr) {}
```

Comment:
Since you are improving the class, let's also remove the `inline` keyword since they do nothing here :)